### ISSUE TITLE
Change rare cases to some cases

### ISSUE THREAD

**Issue by yatil** [w3c/wai-tutorials#130]
**Original Comment:**

> For the 'Labelling Controls' section why does it say that the title attribute is used in 'rare cases'? There may be situations where the <label> cannot be displayed. Better to say 'some cases'.
